### PR TITLE
Fix some whitespace issues

### PR DESCRIPTION
**Pull request by UniQP**

### CODE REVIEW

File: main/src/cgeo/geocaching/enumerations/LogType.java [hunk 1]
```diff
@@ -15,158 +15,158 @@
...
+    SUBMIT_FOR_REVIEW(76, "76", "submit for review", "", R.string.log_submit_for_review),
+    VISIT(1001, "75", "visit", "", R.string.log_tb_visit, R.drawable.mark_green),
+    WEBCAM_PHOTO_TAKEN(11, "11", "webcam photo taken", "", R.string.log_webcam, R.drawable.mark_green, R.drawable.marker_found_offline),
+    ANNOUNCEMENT(74, "74", "announcement", "", R.string.log_announcement),
+    MOVE_COLLECTION(69, "69", "unused_collection", "", R.string.log_movecollection),
+    MOVE_INVENTORY(70, "70", "unused_inventory", "", R.string.log_moveinventory),
+    RETRACT(25, "25", "retract listing", "", R.string.log_retractlisting),
+    MARKED_MISSING(16, "16", "marked missing", "", R.string.log_marked_missing, R.drawable.mark_red),
+    OC_TEAM_COMMENT(83, null, "X1", "OC Team comment", R.string.log_oc_team_comment),
+    UNKNOWN(0, "unknown", "", "", R.string.err_unknown, R.drawable.mark_red); // LogType
```
Comment (SammysHP) [inline main/src/cgeo/geocaching/enumerations/LogType.java]: Hmm… what's with this comment? (I know, it was there already before this commit.)

File: main/src/cgeo/geocaching/connector/gc/GCLogin.java [hunk 2]
```diff
@@ -234,7 +234,7 @@ private boolean switchToEnglish(final String previousPage) {
                 Log.e("Failed to find English language selector");
             }
             // switch to english
-            final Parameters params = new Parameters("__EVENTTARGET", paramEnglish, 
+            final Parameters params = new Parameters("__EVENTTARGET", paramEnglish,
```
Comment (SammysHP) [inline main/src/cgeo/geocaching/connector/gc/GCLogin.java]: Can be merged with next line.

**Comment by UniQP** [comment]
@SammysHP: I adapted the commits according to your suggestions (and fixed an additional typo).

**Comment by samueltardieu** [comment]
Merged, thanks.

**Pull request merged**